autoload/OmniSharp.vim

```diff
@@ -1,3 +1,8 @@
+" Ensure python is supported.
+if !has('python')
+  finish
```

Comment:
If I do that, it acts like a `throw` for some reason. It would cause cascading failures in other dependent bundles.